src/index.js

```diff
@@ -294,6 +294,34 @@ const userEvent = {
     element.addEventListener("blur", fireChangeEvent);
   },
 
+  upload(element, fileOrFiles, init) {
```

Comment:
Would it not make sense to allow configurability of both events?

```suggestion
  upload(element, fileOrFiles, {clickInit, changeInit}) {
```

Reply:
Yep, looks reasonable but here I'm only following `clickElement` function API. It would require breaking changes to all functions which use `clickElement` then. Is it desirable?

Reply:
I think the `clickElement` function is fine as is. No need to change it. It just seems odd to me that we'd allow you to initialize the `click` event and not the `change` event (which is actually what people will be thinking about when using this method).

Reply:
Now I've catched what you mean, thanks for guiding 😄